admin/admin-functions.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * The admin-specific functionality of the plugin.
+ *
+ * @link       https://nataliesmith.dev
+ * @since      1.0.0
+ * @package    Poetwp
+ * @subpackage Poetwp/admin
+ */
+
+/**
+ * Add custom meta boxes to the poem post type
+ */
+function poetwp_add_meta_boxes() {
+	add_meta_box( 'poem_details', 'Poem Details', 'poetwp_poem_details_callback', 'poem', 'normal', 'default' );
```

Comment:
It's worth building in i18n from the beginning everywhere you have a string that's visible to the user, so here, the `'Poem Details'` would be written as `__( 'Poem Details', 'poetwp' )`. The text domain should match the plugin slug. You can then also configure PHPCS with the text domain to check, and it will look for missing or incorrect text domains on i18n'd strings.

Reply:
Noted #16